src/config-file.c

```diff
@@ -44,6 +45,7 @@ void config_file_free(struct config *config)
         g_free(config->auth_token);
         g_free(config->gateway_token);
         g_free(config->bundle_download_location);
+        g_free(config->post_update_reboot);
```

Comment:
post_update_reboot is not a pointer. Nothing to free. Remove this line

Reply:
yes of course, thanks.